src/main/scala/com/wavesplatform/consensus/GeneratingBalanceProvider.scala

```diff
@@ -23,8 +24,10 @@ object GeneratingBalanceProvider {
         .get(BlockchainFeatures.SmallerMinimalGeneratingBalance.id)
         .exists(height >= _) && effectiveBalance >= MinimalEffectiveBalanceForGenerator2
 
-  def balance(blockchain: Blockchain, fs: FunctionalitySettings, height: Int, account: Address): Long = {
+  def balance(blockchain: Blockchain, fs: FunctionalitySettings, account: Address, block: ByteStr = ByteStr.empty): Long = {
```

Comment:
Can you rename `block` to `blockId` here? `block` suggests something of type `Block`

Reply:
Ok